### PR TITLE
Nicole/Random k mutants guidance

### PR DESCRIPTION
**Pull request by nicoleraenair**

### CODE REVIEW

**Comment by nicoleraenair** [comment]
Campaign with MutationGuidance:
plot before
Screen Shot 2022-06-10 at 3 27 27 PM

Campaign with RandomMutationGuidance (where k = 50)
Screen Shot 2022-06-10 at 3 47 06 PM
Screen Shot 2022-06-10 at 3 46 40 PM

File: src/main/java/cmu/pasta/mu2/fuzz/RandomMutationGuidance.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Override
+    public ArraySet filterMutants() {
+
+        ArrayList<Integer> mutantList = new ArrayList<Integer>();
+        int numMutants = 0;
+        int size = runMutants.size();
+        int n = 0;
+
```
Comment (rohanpadhye) [inline src/main/java/cmu/pasta/mu2/fuzz/RandomMutationGuidance.java:31]: Could you please add a comment about what this loop does (e.g. "collect mutants into a list")?

**Comment by rohanpadhye** [comment]
Good job with the implementation and evaluation! I've added some minor comments about code quality, and we will also have to refactor this with changes to the API as discussed in #54.

**Review by rohanpadhye — APPROVED** [review]
Looks good, thanks!

**Pull request merged**